Specifications.md

```diff
@@ -5,43 +5,43 @@
+| [Jakarta Mail](https://github.com/eclipse-ee4j/mail-spec) | [2.0](https://jakarta.ee/specifications/mail/2.0/) | [Eclipse Jakarta Mail 2.0.0](https://eclipse-ee4j.github.io/mail/) |
+| [Jakarta Managed Beans](https://github.com/eclipse-ee4j/jakartaee-platform) | [2.0](https://jakarta.ee/specifications/managedbeans/2.0/) | [Eclipse Glassfish 6.0](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
 | [Jakarta Management](https://github.com/eclipse-ee4j/management-api) | [1.1](https://jakarta.ee/specifications/management/1.1/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
-| [Jakarta Messaging](https://github.com/eclipse-ee4j/jms-api) | [2.0](https://jakarta.ee/specifications/messaging/2.0/) | [Eclipse OpenMQ 5.1.4](https://eclipse-ee4j.github.io/openmq/certifications/jakarta-messaging/2.0/TCK-Results) |
-| [Jakarta Persistence](https://github.com/eclipse-ee4j/jpa-api) | [2.2](https://jakarta.ee/specifications/persistence/2.2/) | [EclipseLink 2.7.4](https://www.eclipse.org/eclipselink) |
-| [Jakarta RESTful Web Services](https://github.com/eclipse-ee4j/jaxrs-api) | [2.1](https://jakarta.ee/specifications/restful-ws/2.1/) | [Eclipse Jersey 2.28](https://projects.eclipse.org/projects/ee4j.jersey) |
-| [Jakarta Security](https://github.com/eclipse-ee4j/security-api) | [1.0](https://jakarta.ee/specifications/security/1.0/) | [Eclipse Soteria 1.0.1](https://eclipse-ee4j.github.io/soteria) |
-| [Jakarta Server Faces](https://github.com/eclipse-ee4j/faces-api) | [2.3](https://jakarta.ee/specifications/faces/2.3/) | [Mojarra 2.3.13](https://eclipse-ee4j.github.io/mojarra) |
-| [Jakarta Server Pages](https://github.com/eclipse-ee4j/jsp-api) | [2.3](https://jakarta.ee/specifications/pages/2.3/) | [Eclipse GlassFish 5.1](https://eclipse-ee4j.github.io/glassfish/) |
-| [Jakarta Servlet](https://github.com/eclipse-ee4j/servlet-api) | [4.0](https://jakarta.ee/specifications/servlet/4.0/) | [Eclipse GlassFish 5.1](https://eclipse-ee4j.github.io/glassfish/) |
-| [Jakarta Standard Tag Library](https://github.com/eclipse-ee4j/jstl-api) | [1.2](https://jakarta.ee/specifications/tags/1.2/) | [Eclipse GlassFish 5.1](https://eclipse-ee4j.github.io/glassfish/) |
-| [Jakarta Transactions](https://github.com/eclipse-ee4j/jta-api) | [1.3](https://jakarta.ee/specifications/transactions/1.3/) | [Eclipse GlassFish 5.1.0](https://eclipse-ee4j.github.io/glassfish/) |
-| [Jakarta WebSocket](https://github.com/eclipse-ee4j/websocket-api) | [1.1](https://jakarta.ee/specifications/websocket/1.1/) | [Eclipse GlassFish 5.1](https://eclipse-ee4j.github.io/glassfish/) |
+| [Jakarta Messaging](https://github.com/eclipse-ee4j/jms-api) | [3.0](https://jakarta.ee/specifications/messaging/3.0/) | [Eclipse OpenMQ 6.0.0](https://jakarta.oss.sonatype.org/content/repositories/staging/org/glassfish/mq/mq-distribution/) |
```

Comment:
It is better to use 
"https://projects.eclipse.org/projects/ee4j.openmq"
or
"https://github.com/eclipse-ee4j/openmq"
for the implementation URL.

Reply:
> It is better to use
> "https://projects.eclipse.org/projects/ee4j.openmq"
> or
> "https://github.com/eclipse-ee4j/openmq"
> for the implementation URL.

Thanks, @kazumura.  I wanted to use a link that had an actual binary download and this is what was used for the CCR.  How about this one instead?  https://github.com/eclipse-ee4j/openmq/releases/tag/6.0.0-RELEASE  The downloadable zip file has the binaries for the 6.0 release.  I'll update this accordingly.  Thanks.